js/views/callinfoview.js

```diff
@@ -28,6 +28,8 @@
 	OCA.SpreedMe = OCA.SpreedMe || {};
 	OCA.SpreedMe.Views = OCA.SpreedMe.Views || {};
 
+	var uiChannel = Backbone.Radio.channel('ui');
```

Comment:
ESlint fails:
> /drone/src/github.com/nextcloud/spreed/js/views/callinfoview.js
> 31:6 error 'uiChannel' is defined but never used no-unused-vars

Reply:
Must be a rebase error, I don't recall adding this line! Let me remove it!